### ISSUE TITLE
Does not support servers/url option in openApi v3

### ISSUE THREAD

**Issue by williamzhao87** [seriousme/fastify-openapi-glue#46]
I noticed that the generated routes ignore the servers/url parameter (this is the equivalent of basePath in openApi v2) defined in the specs yaml file for openApi v3.

E.g.
```
openapi: '3.0.0'
servers:
  - url: /api/v3
paths:
  /info:
    get:
    ....
```
Expected: 
Route generated with path /api/v3/info

Actual:
Route generated with path /info

Just wanted to check if this is intended, or just not yet supported.

**Comment by seriousme** [comment]
Hi, 

thanks for asking.

When creating the parser for v3 I decided to skip the `servers/url` config as the parameter includes a hostname as well and allows for multiple values (see https://swagger.io/docs/specification/api-host-and-base-path/)

Thinking about it today I guess that it might be convenient to have a prefix parameter so one can specify something like:

```javascript
const openapiGlue = require("fastify-openapi-glue");

const options = {
  specification: `${__dirname}/petstore-swagger.v2.json`,
  service: `${__dirname}/service.js`,
  prefix:"v1"
};

fastify.register(openapiGlue, options);
```

An alternative is to use the route prefixing mechanism in fastify (see https://github.com/fastify/fastify/blob/master/docs/Routes.md#route-prefixing) which offers quite some flexibility.

What do you think ?

Kind regards,
Hans

**Comment by williamzhao87** [comment]
The prefix option in the options object would be very useful, I think!

For now, I will try the alternative method to see if it works.  Thanks!

**Comment by seriousme** [comment]
Using the 'servers/url' in special cases only makes it less predictable to users I think.
I created the extra option in https://github.com/seriousme/fastify-openapi-glue/tree/add-prefix-option and added tests (we're still at 100% coverage  ;-))
I need to wait for CI to complete and then I can merge and release the new version.

**Comment by seriousme** [comment]
Ok, merged (#47) and published as version 1.1.0 on NPM

**Comment by williamzhao87** [comment]
Thanks!
